src/global_logic.js

```diff
@@ -34,8 +35,9 @@ module.exports.isCosmos = function (arm) {
 };
 
 module.exports.isDarkOpus = function (arm) {
-    return (arm.skill1 != undefined && arm.skill1 == "normalKamuiL") ||
-        (arm.skill1 != undefined && arm.skill1 == "magnaKamuiL");
+    return darkOpusWeapons.some(function (value) {
+        return arm.id == value;
+    });
```

Comment:
I am not sure this, if Babel allow more simple way
- darkOpusWeapons.some(value => arm.id == value);
- darkOpusWeapons.includes(value)